gameswap-service/pom.xml

```diff
@@ -91,6 +91,12 @@
             <version>${dropwizard.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.6.1</version>
+            <scope>test</scope>
```

Comment:
it was just a preference. Probably a good thing to remove unnecessary dependencies